tests/e2e/core-tests/specs/merchant/wp-admin-product-new.test.js

```diff
@@ -19,6 +19,7 @@ const {
 const config = require( 'config' );
 
 const simpleProductName = config.get( 'products.simple.name' );
+const simpleProductPrice = config.get( 'products.simple.price' );
```

Comment:
@zhongruige Any projects using the existing packages with a local `default.json` will not have the price entry in their config. To make this PR backward compatible use a default price of `9.99` when it's not found in the config.

Looking at the read me files, I see that we are missing a [`Test Variables`](https://github.com/woocommerce/woocommerce/tree/master/tests/e2e#test-variables) section in the core tests readme. This could be handled in a follow up PR.

Reply:
That makes sense--thanks for pointing out the backwards compatibility. I'll make that change.

Reply:
Just to follow up on this, I'll update the tests to have this check for the backwards compatibility:

```
const simpleProductPrice = config.has('products.simple.price') ? config.get('products.simple.price') : '9.99';
```